c/planarityTestGraphFunctionality.c

```diff
@@ -226,16 +310,16 @@ int transformString(graphP theGraph, char *inputStr)
 	TEST ALL GRAPHS IN .G6
 */
 
-int testAllGraphs(graphP theGraph, char command, char *inputStr, strOrFileP testOutput)
+int testAllGraphs(graphP theGraph, char command, char *inputStr, testAllStatsP stats)
```

Comment:
I really like that you did this as a struct so we can add other things later, if we choose to, like some of the extra stats we compute when testing randomly generated graphs.